app/controllers/SecureMessagingController.scala

```diff
@@ -34,24 +38,50 @@ class SecureMessagingController @Inject()(
   messageConnector: SecureMessageFrontendConnector,
   mcc: MessagesControllerComponents,
   inbox_wrapper: inbox_wrapper,
-  conversation_wrapper: conversation_wrapper
+  conversation_wrapper: conversation_wrapper,
+  reply_result: reply_result
```

Comment:
Change name to 'reply_wrapper' to keep consistency with other two view names

Reply:
:+1: